egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -26,6 +26,7 @@ SECONDS=0
 stage=1          # Processes starts from the specified stage.
 stop_stage=10    # Processes is stopped at the specified stage.
 ngpu=1           # The number of gpus ("0" uses cpu, otherwise use gpu).
+num_nodes=1      # The number of nodes
```

Comment:
We might want to unify the naming. Why don't you use `nnode`?

Reply:
Maybe num_* is more clear than n*, in my recent impression. `nnodes` is really difficult.
I would rather change ngpu to num_gpus.